packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2437,6 +2441,108 @@ module.exports = function (log, config, oauthdb) {
     });
   };
 
+  Mailer.prototype.syncPrimaryReminderEmail = function (message) {
```

Comment:
@LZoog You could probably follow the pattern [here](https://github.com/mozilla/fxa/blob/58d02e213cebd5ef3f52c81a679acb3e52c7802f/packages/fxa-auth-server/lib/senders/email.js#L632) and dynamically build out these.